src/schema/campaigns.ts

```diff
@@ -85,6 +87,13 @@ export const typeDefs = /* GraphQL */ `
     max: Int!
   }
 
+  type UserCampaignData {
```

Comment:
```suggestion
  type UserCampaignStats {
```

Data is generic terms, maybe stats or analytics is better?

Reply:
Yeah, my mind was battling which one to use 😅